extensions.tf

```diff
@@ -37,6 +37,7 @@ module "vm-bootstrap" {
   run_xdr_collector  = var.run_xdr_collector
   run_xdr_agent      = var.run_xdr_agent
   run_cis            = var.run_cis
+  xdr_tags           = join(",", [for key, value in var.tags : "${key}=${value}"])
```

Comment:
just my thoughts, we move this to a locals variable
`locals {
xdr_tags           = join(",", [for key, value in var.tags : "${key}=${value}"])
}`
then here we say:
`xdr_tags           = local.xdr_tags`

does that look nice :)